doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -714,29 +714,199 @@ general for HDDs, the bluestore throttle values are expected to be higher when
+values like shown below:
+
+.. prompt:: bash #
+
+  ceph config dump
+
+::
+
+  WHO     MASK  LEVEL  OPTION                            VALUE       RO
+  osd.0         basic  osd_mclock_max_capacity_iops_hdd  379.197568    
+  osd.1         basic  osd_mclock_max_capacity_iops_hdd  400.903575    
+  osd.2         basic  osd_mclock_max_capacity_iops_hdd  398.303428    
+  osd.3         basic  osd_mclock_max_capacity_iops_hdd  419.035854    
+
+The config global is set using:
+
+.. prompt:: bash #
+
+  ceph config set global osd_mclock_max_capacity_iops_hdd 370
+
+The config dump now looks like:
+
+.. prompt:: bash #
+
+  ceph config dump
+
+::
+
+  WHO     MASK  LEVEL  OPTION                            VALUE       RO
+  global        basic  osd_mclock_max_capacity_iops_hdd  370.000000    
+  osd.0         basic  osd_mclock_max_capacity_iops_hdd  379.197568    
+  osd.1         basic  osd_mclock_max_capacity_iops_hdd  400.903575    
+  osd.2         basic  osd_mclock_max_capacity_iops_hdd  398.303428    
+  osd.3         basic  osd_mclock_max_capacity_iops_hdd  419.035854    
+
+But individual values for the OSDs are still not overridden due to the way
+the config subsystem is designed. See :ref:`ceph-conf-settings` for more details.
+For e.g., osd.0 still shows its individual value in-effect shown below:
+
+.. prompt:: bash #
+
+  ceph config show osd.0 | grep osd_mclock
+
+::
+
+  osd_mclock_max_capacity_iops_hdd                 379.197568                               mon
+
+For the global setting to come into effect, the individually scoped values
+must be removed like shown below:
+
+.. prompt:: bash #
+
+  ceph config rm osd.0 osd_mclock_max_capacity_iops_hdd
+
+.. prompt:: bash #
+
+  ceph config rm osd.1 osd_mclock_max_capacity_iops_hdd
+
+.. prompt:: bash #
+
+  ceph config rm osd.2 osd_mclock_max_capacity_iops_hdd
+
+.. prompt:: bash #
+
+  ceph config rm osd.3 osd_mclock_max_capacity_iops_hdd
+
+The config dump now looks like:
+
+.. prompt:: bash #
+
+  ceph config dump
+
+::
+
+  WHO     MASK  LEVEL  OPTION                            VALUE       RO
+  global        basic  osd_mclock_max_capacity_iops_hdd  370.000000    
+
+After removing the individual settings, the global setting automatically comes
+into effect for all the OSDs as shown below:
+
+.. prompt:: bash #
+
+  ceph config show osd.0 | grep osd_mclock_max_capacity_iops_hdd
+
+::
+
+  osd_mclock_max_capacity_iops_hdd                 370.000000                               mon
+
+.. prompt:: bash #
+
+  ceph config show osd.1 | grep osd_mclock_max_capacity_iops_hdd
+
+::
+
+  osd_mclock_max_capacity_iops_hdd                 370.000000                               mon
+
+.. prompt:: bash #
+
+  ceph config show osd.2 | grep osd_mclock_max_capacity_iops_hdd
+
+::
+
+  osd_mclock_max_capacity_iops_hdd                 370.000000                               mon
+
+.. prompt:: bash #
+
+  ceph config show osd.3 | grep osd_mclock_max_capacity_iops_hdd
+
```

Comment:
Again, we don't need to repeat this four times.  Condense to a single prompt block and show the effect of removing  one per-OSD scope setting while the others are not affected.

Reply:
Done.